File: mentor-on-demand-app-client/src/app/models/user-profile.ts
import { last } from 'rxjs/operators';

export class UserProfile {

    id: number;
    firstName: string;
    lastName: string;
    totalYearsExp: number;
    contactNumber: string;
    linkedInUrl: string;
    introduction: string;

    constructor(id: number
        , firstName: string
        , lastName: string
        , totalYearsExp: number
        , contactNumber: string
        , linkedInUrl: string
        , introduction: string) {

            this.id = id;
            this.firstName  = firstName;
            this.lastName = lastName;
            this.totalYearsExp = totalYearsExp;
            this.contactNumber = contactNumber;
            this.linkedInUrl = linkedInUrl;
            this.introduction = introduction;
    }

}

File: mentor-on-demand-system-user-ms/src/main/java/com/ibm/mods/user/model/UserProfile.java
package com.ibm.mods.user.model;

public class UserProfile {
	
	private long id;
	private String firstName;
	private String lastName;
	private int totalYearsExp;
	private String contactNumber;
	private String linkedInUrl;
	private String introduction;
	
	public long getId() {
		return id;
	}
	public void setId(long id) {
		this.id = id;
	}
	public String getFirstName() {
		return firstName;
	}
	public void setFirstName(String firstName) {
		this.firstName = firstName;
	}
	public String getLastName() {
		return lastName;
	}
	public void setLastName(String lastName) {
		this.lastName = lastName;
	}
	public int getTotalYearsExp() {
		return totalYearsExp;
	}
	public void setTotalYearsExp(int totalYearsExp) {
		this.totalYearsExp = totalYearsExp;
	}
	public String getContactNumber() {
		return contactNumber;
	}
	public void setContactNumber(String contactNumber) {
		this.contactNumber = contactNumber;
	}
	public String getLinkedInUrl() {
		return linkedInUrl;
	}
	public void setLinkedInUrl(String linkedInUrl) {
		this.linkedInUrl = linkedInUrl;
	}
	public String getIntroduction() {
		return introduction;
	}
	public void setIntroduction(String introduction) {
		this.introduction = introduction;
	}

}
